### ISSUE TITLE
How to change .wakatime.log,db,data positions

### ISSUE THREAD

**Issue by koumakpet** [wakatime/wakatime#201]
I'm using linux an wakatime files are cluttering my home directory, is it possible to move these files somewhere else?
Let's say I have a floder like `~/config/wakatime/` and I'd like to put all the wakatime files in it (*.wakatime.log .wakatime.db .wakatime.data*) now I know I probably can't move the *.wakatime.cfg* but is there a setting that could change the default locations of these files?

**Comment by gandarez** [comment]
No you can't move wakatime system files because all the plugins rely on `$HOME` directory.

**Comment by koumakpet** [comment]
Hi, I noticed you've changed your answer I haven't looked here so sorry it took so long to respond.

But when I tried to create `WAKATIME_HOME` variable with value **/home/koumakpet/.config/wakatime** and put all .wakatime* files in there, I tried to open Visual Studio Code and it instantly asked me for Wakatime API key, when I provided it, it created .wakatime.cfg in my home again.

So i wonder,
1. does WAKATIME_HOME not apply on .wakatime.cfg only on those other 3 files
2. does it not apply for visual studio code specificly?
3. is there a bug?
4. it is also possible I did something wrong but I can't see what, this is the line in my .zshrc :
```
export WAKATIME_HOME=$HOME/.config/wakatime
````

Once again, sorry for the late response
